Forms/MainForm.cs

```diff
@@ -80,15 +93,37 @@ private void LoadDataFromIso()
       this.toggleEnableControls(true);
     }
 
+    private void ExportGridToCsv(DataGridView grid, string fileName, string separator = ";")
```

Comment:
This works well enough for some tables but not for others. The fusions table for instance when exported only lists the IDs of the cards. This would make more sense if its only purpose was to import it back into this program.